.github/workflows/Vulnerability-Scan.yaml

```diff
@@ -207,6 +207,11 @@ jobs:
           fi
           echo "img-repo=$img_repo" >> "$GITHUB_OUTPUT"
 
+      - name: Fetch _releases.json
```

Comment:
you should also have `if: ${{ inputs.create-issue }}`, otherwise it will again fail for reusable workflows